hack/ci-test-e2e.sh

```diff
@@ -57,6 +57,14 @@ bin/test-e2e \
   --e2e.additional-tags="expirationDate=$(date -d '4 hours' --iso=minutes --utc)" \
   --e2e.aws-endpoint-access=PublicAndPrivate \
   --e2e.external-dns-domain=service.ci.hypershift.devcluster.openshift.com \
-  ${REQUEST_SERVING_COMPONENT_PARAMS} | tee /tmp/test_out &
+  ${REQUEST_SERVING_COMPONENT_PARAMS}
+)
+
+declare -a args="$@"
+if [[ "$#" -lt 2 ]]; then
+  args="${default_args[@]}"
+fi
```

Comment:
This is cryptic.  Could do with a comment.

Reply:
Sure thing, Bash is not the best :)